src/mlpack/tests/ann/layer/c_relu.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file tests/ann/layer/c_relu.cpp
+ * @author Marcus Edel
+ * @author Dhawal Arora
+ *
+ * Tests the concatenated relu layer modules.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/methods/ann.hpp>
+
+#include "../../test_catch_tools.hpp"
+#include "../../catch.hpp"
+#include "../../serialization.hpp"
+#include "../ann_test_tools.hpp"
+
+using namespace mlpack;
+
+/**
+ * Basic test of the CReLU function.
+ */
+TEST_CASE("CReLUFunctionTest", "[ANNLayerTest]")
+{
+  const arma::colvec desiredActivations("0 3 0 6 24 \
+                                         2 0 0 0 0 0 \
+                                         100.2 0 1 0 0");
```

Comment:
It seems to me like the last several elements of this can be removed.  The size should be 2x the size of `activationData`.